src/actions/userActions.js

```diff
@@ -135,3 +136,37 @@ export const updateUserAvatarAction = (walletId: string, formData: any) => {
   };
 };
 
+export const labelUserAsLegacyAction = () => {
+  return async (dispatch: Dispatch, getState: GetState, api: Object) => {
```

Comment:
If possible, add the type to the `api` object (we'd need to import the wrapper: `import SDKWrapper from 'services/api';`)

```suggestion
  return async (dispatch: Dispatch, getState: GetState, api: SDKWrapper) => {
```